### DISCUSSION TITLE
Question: Is there a recommended approach to split deployments?

### DISCUSSION THREAD

**Discussion by andrewjroberts** [payloadcms/payload#1220] [Q&A] [answered]
Hi all,

Is there a recommended approach to split deployments, i.e. a separate deployment for Admin and API?

**Approaches we've considered**:

- Use the `disable` properties for Admin and GraphQL (but we weren't sure how to disable the REST API...)
- Disabling [top level access](https://payloadcms.com/docs/access-control/overview) on internal deployment

**Use case**:
My team wants to externally expose the Admin panel, but keep the API internal—because our app hits a service that pulls from the CMS internally (+ Cloud CDN caching).

**Comment by jmikrut** [comment, marked as answer]
Yes, you can—Payload's admin UI gets built to the `build` folder, and if you place all of those files on a separate host, you can deploy them separately. Just make sure you have the proper `serverURL` set, pointing at the backend API location, when you build the admin panel files. 👍
